lib/ansible/modules/cloud/azure/azure_rm_cosmosdbaccount_facts.py

```diff
@@ -339,10 +339,10 @@
                              q3dpJxJga76h9BZkK2BJJrDzSO6DDn6yKads017OZBZ1YZWyq1cW4iuvA=="
         tags:
             description:
-                - Tags
+                - Tags assigned to the resource. Dictionary of string:string pairs.
```

Comment:
I would put the quotes in here as well, like this:

```suggestion
                - Tags assigned to the resource. Dictionary of "string":"string" pairs.
```